src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
```

Comment:
better to name files and folder as component name
![Screenshot from 2023-08-26 17-14-42](https://github.com/mate-academy/react_todo-app-loading-todos/assets/33029864/05af284f-9ec7-4311-8247-b1330c2a8e70)